backend/src/v5/models/jobs.js

```diff
@@ -30,7 +32,11 @@ Jobs.addDefaultJobs = async (teamspace) => {
 };
 
 Jobs.assignUserToJob = async (teamspace, job, username) => {
-	await db.updateOne(teamspace, COL_NAME, { _id: job }, { $push: { users: username } });
+	await updateOne(teamspace, { _id: job }, { $push: { users: username } });
+};
+
+Jobs.removeUserFromJobs = async (teamspace, userToRemove) => {
+	await updateMany(teamspace, { }, { $pull: { users: userToRemove } });
```

Comment:
Probably better to filter out any jobs that doesn't have the user in it.

```suggestion
	await updateMany(teamspace, { users: userToRemove }, { $pull: { users: userToRemove } });
```
```suggestion
	await updateMany(teamspace, { }, { $pull: { users: userToRemove } });
```